src/EnergyPlus/Autosizing/Base.cc

```diff
@@ -718,6 +718,85 @@ Real64 BaseSizer::setCoolCoilInletHumRatForZoneEqSizing(Real64 const outAirFrac,
     return coilInHumRat;
 }
 
+void BaseSizer::calcCoilWaterFlowRates(EnergyPlusData &state,
```

Comment:
There is a epluszsz.csv (zone), eplusssz (system), and now a epluspsz.csv (plant). The psz file reports the demand side component water flow rate that ultimately sizes the plant. I needed this to make sure I was summing the time step water flow rate correctly and then it seemed it was a good output for user inspection. This branch only sums the water coils. There are other coil types that add to the plant load that will eventually need to be included. e.g., water source HPs, LoadProfile, etc.